{
  final NamingException exception=new NamingException(message);
  exception.setRootCause(cause);
  return exception;
}
